Terminal.Gui/View/Adornment/Adornment.cs

```diff
@@ -33,7 +33,8 @@ public Adornment ()
     /// <summary>Gets the rectangle that describes the inner area of the Adornment. The Location is always (0,0).</summary>
     public override Rectangle Bounds
     {
-        get => Thickness?.GetInside (new Rectangle (Point.Empty, Frame.Size)) ?? new Rectangle (Point.Empty, Frame.Size);
+        get => Thickness?.GetInside (new (Point.Empty, Frame.Size)) ?? new Rectangle (Point.Empty, Frame.Size);
+        // QUESTION: So why even have a setter then?
```

Comment:
To find code that was doing it.

Reply:
But if there were no setter, it'd be a compiler error anyway.

Or was that a consequence of something that was being done by reflection?